### ISSUE TITLE
EXT1 and USB-UART1 are too close

### ISSUE THREAD

**Issue by Miceuz** [OLIMEX/ESP32-EVB#12]
It is impossible to use 40 pin IDC shrouded header for EXT1 as USB connector is too close.

![image](https://user-images.githubusercontent.com/533263/34034158-f31113b0-e185-11e7-98e9-44d18db1e2ff.png)

It is possible to trick it into semi-usable configuration, but pin numbering is screwed up in this case and ultra-thin usb connector has to be used.
 
![image](https://user-images.githubusercontent.com/533263/34034355-c7b9fde8-e186-11e7-82ee-ead08691f720.png)

So basically, the only workaround is to use 34 pin connector and loose access to +5V and +3.3V.

Otherwise, thank you for this board. We are using it to build an electronic lock in our hackerspace.

**Comment by DanKoloff** [comment]
This is not the connector we had in mind during the design. The schematic of the board says what is the component that we had in mind while designing these pinholes - it is HN2X20 (and the NA is short from Not Assembled). Search for "HN2x20 connector" to see such products.

The pinholes were meant to be used with male connectors like these: https://www.olimex.com/Products/Components/Connectors/MALE-PAV16X-2x05/ - they have a plastic base that would lift it above the micro USB connector and would allow connecting a female.
